index.md

```diff
@@ -54,21 +53,18 @@ deck: "Easy, compliant customer feedback for federal agencies."
       Feedback as a Service
     </strong>
     <p>
-      Touchpoints a FIPS Moderate-rated web application, deployed to cloud.gov.
+      Touchpoints is a FIPS Moderate-rated web application, deployed to cloud.gov.
       Spend more time on feedback and less time on tools
       by leveraging Touchpoints to collect customer feedback.
     </p>
   </div>
   <div class="tablet:grid-col">
     <strong>
-      PRA-friendly A-11 Form
+      Streamlined Reporting
```

Comment:
I could see this 2nd column moved to the column 3 instead.

Reply:
I updated the column ordering. Now "Reporting" is the 3rd column, instead of the 2nd.